_posts/blog/2016-10-20-reactive_frustrations_1.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: "Reactive Frustrations Part 1"
```

Comment:
I would leave this as "Reactive Frustrations" and then put "Part 1" when you write the Part 2.